src/Servers/Kestrel/shared/KnownHeaders.cs

```diff
@@ -20,33 +21,33 @@ public class KnownHeaders
     public static readonly KnownHeader[] ResponseTrailers;
     public static readonly string[] InternalHeaderAccessors = new[]
     {
-            HeaderNames.Allow,
-            HeaderNames.AltSvc,
-            HeaderNames.TransferEncoding,
-            HeaderNames.ContentLength,
-            HeaderNames.Connection,
-            HeaderNames.Scheme,
-            HeaderNames.Path,
-            HeaderNames.Method,
-            HeaderNames.Authority,
-            HeaderNames.Host,
-        };
+        HeaderNames.Allow,
+        HeaderNames.AltSvc,
+        HeaderNames.TransferEncoding,
+        HeaderNames.ContentLength,
+        HeaderNames.Connection,
+        HeaderNames.Scheme,
+        HeaderNames.Path,
+        HeaderNames.Method,
+        HeaderNames.Authority,
+        HeaderNames.Host,
+    };
```

Comment:
@pranavkm Array initialize indenting was messed up when file scoped namespaces were introduced.

Reply:
Yup. I noticed it too. There's also some nesting for lambdas that are messed up. I'll get a PR to clean it up.

FYI @davidwengier in case you wanted to have a look at it (or perhaps this is resolved as part of your trivia fix).

Reply:
Sadly, its a long standing limitation of the formatter, that it doesn't touch collection, object and array initializers. Unfortunately when file scoped namespaces come along and de-indent everything, that fact becomes very visible. The issue is tracked in https://github.com/dotnet/roslyn/issues/8269